### ISSUE TITLE
Style accuracy mismatch  85.6%(Table 1, from the paper) Vs 88.40% (using https://github.com/luofuli/DualRL#step-1-pre-train-classifier and https://github.com/luofuli/DualRL/tree/master/outputs/yelp/DualRL)

### ISSUE THREAD

**Issue by gourango01** [luofuli/DualRL#21]
Thanks, for having a separate section for **reproducibility** (https://github.com/luofuli/DualRL#reproducibility.)  on this great work.
When I am using a **_pre-train classifier_** (same as https://github.com/luofuli/DualRL#step-1-pre-train-classifier) for evaluating the style accuracy, I am getting 88.40% for the **_yelp dataset_** on **_DUAL RL output_**(https://github.com/luofuli/DualRL/tree/master/outputs/yelp/DualRL).
But you have reported 85.6% only. So what could be the reason for this difference? Am I missing something?

**Comment by luofuli** [comment]
Since the accuracy score is evaluated by a pre-trained classifier which may be dependent on many factors like random seeds, systems (GPU version, Cuda version) and so on. So I think the accuracy scores of our models and other baselines may not be fully reimplemented. **That's why I release all the outputs of our models and all other baselines**. In this way, any other researchers can recompute the accuracy score using your own trained classifier to have a fair comparison. And I believe that **the relative order of each system's accuracy score** can the same as the reported in our papers.
